tensorrt_llm/_torch/custom_ops/cute_dsl_custom_ops.py

```
@@ -2789,3 +2793,308 @@ def _(
+            [runner],
+            runner.__class__.tuning_config,
+            inputs,
+        )
+        return runner(inputs, tactic=best_tactic)
+
+    @torch.library.register_fake(
+        "trtllm::cute_dsl_fp8_blockwise_grouped_gemm_blackwell")
+    def _(
+        input: torch.Tensor,
+        weight: torch.Tensor,
+        input_scale: torch.Tensor,
```

Ruff (0.15.0)

[warning] Unused function argument: `input_scale`

(ARG001)